src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -17,7 +17,7 @@
 public final class BucketUtilsTest extends GATKBaseTest {
 
     static {
-        BucketUtils.setGlobalNIODefaultOptions();
+        BucketUtils.setGlobalNIODefaultOptions(20,"");
```

Comment:
Can you get the defaults from the GATKConfig instead of hardcoding them here?

Reply:
Yes, done.